gradle-plugin/src/main/java/com/contrastsecurity/gradle/plugin/ContrastGradlePlugin.java

```diff
@@ -11,11 +25,156 @@ public class ContrastGradlePlugin implements Plugin<Project> {
 
   public void apply(final Project target) {
 
-    ContrastConfigurationExtension extension =
+    final String agentPath =
```

Comment:
add comment that we're only using this as a target path. maybe that's kinda overkill but 🤷